### PR TITLE
Add image to slide

### PR DESCRIPTION
**Pull request by adkron**
This was a squashed commit and I kept the original commit messages
because I think they shed some light on the process.

Amos King @adkron <amos@binarynoggin.com>

------------------------------

This is the smallest amount of change I could get to add a picture to a
slide. I put text on the slide too because I think that is a normal use
case and they should work flawlessly together.

Amos King @adkron <amos@binarynoggin.com>

Add image xml

This adds all xml needed for an image except for content type.

Amos King @adkron <amos@binarynoggin.com>

Add image file and content type

All the image content types have the same name as the extension except
for the short hand `jpg` file path. This is why I only made a single entry
in the `TYPES` and allow the rest to default. This gives us a nice
extension point for the future. It would be nice to detect the mime type
instead of relying on the extension except that we can't reliably do
that on every OS to my knowledge.

Amos King @adkron <amos@binarynoggin.com>

### CODE REVIEW

**Comment by adkron** [comment]
There are a lot of shared elements here that are in drawingML, but the wrong namespace. I think that a good refactor would be to allow a namespace to be passed in to override the default.

File: spec/shapes/bounds_spec.rb [hunk 1]
```diff
@@ -18,13 +18,13 @@
 
   describe "with a bottom bound that is above the top" do
```
Comment (kobsy) [inline spec/shapes/bounds_spec.rb]: Seems like you reused the same test title; is this testing the horizontal bounds?

**Review by kobsy — COMMENTED** [review]
Looks good! Images are nuts in OpenXML.

File: lib/openxml/pptx/elements/non_visual_picture_properties.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+module OpenXml
+  module Pptx
+    module Elements
+      class NonVisualPicturProperties < OpenXml::Container
+        namespace :p
```
Comment (robrieba) [inline lib/openxml/pptx/elements/non_visual_picture_properties.rb]: Did you want this to be `NonVisualPictureProperties`?

**Review by robrieba — COMMENTED** [review]
Looks good to me, other than the possible typo: `NonVisualPicturProperties`

**Review by boblail — APPROVED** [review]
Looks good! 👍

**Pull request merged**